### DISCUSSION TITLE
能否把它编译成一个或多个deb包，供其他电脑直接安装

### DISCUSSION THREAD

**Discussion by zhangyiming748** [linuxdeepin/developer-center#9059] [Development Questions | 开发答疑]
能否把它编译成一个或多个deb包，在同样硬件和系统的电脑上省去重复编译安装的过程？

**Comment by zhangyiming748** [comment]
原来这不是单个仓库的issue  
我说的软件是[deepin-unioncode](https://github.com/linuxdeepin/deepin-unioncode)

**Comment by BLumia** [comment]
分发的是不是 deb 包和它是否可跨发行版直接安装使用是没有关系的，目前仓库里的实际就是 deb 包版，但你应该无法跨发行版使用现成的 deb 包。

暂时不清楚 IDE 项目组是否打算对主流发行版都提供现成的二进制包，目前的话如果是非 deepin 环境，暂时还是建议手动编译安装吧。
